ext/spl/spl_heap.c

```diff
@@ -1135,8 +1135,8 @@ ZEND_BEGIN_ARG_INFO(arginfo_heap_insert, 0)
 ZEND_END_ARG_INFO()
 
 ZEND_BEGIN_ARG_INFO(arginfo_heap_compare, 0)
-	ZEND_ARG_INFO(0, a)
-	ZEND_ARG_INFO(0, b)
+	ZEND_ARG_INFO(0, value1)
+	ZEND_ARG_INFO(0, value2)
```

Comment:
Should update https://github.com/php/php-src/blob/0fe767d559cb52f38433e0e1c15c9f299058cca8/ext/spl/spl_heap.c#L857 and friends.